Nos/Models/ReportCategory.swift

```diff
@@ -2,16 +2,16 @@ import Foundation
 
 /// A model for potential reasons why something might be reported.
 struct ReportCategory: Identifiable, Equatable {
-
+    
```

Comment:
I wonder if Xcode is adding these trailing spaces; check Xcode > Settings > Text Editing > Editing and see if "Automatically trim trailing whitespace" is on as well as the sub-option for "Including whitespace-only lines".
<img width="942" alt="Screenshot 2024-04-25 at 8 59 17 AM" src="https://github.com/planetary-social/nos/assets/59564/4f746396-99af-41ba-a6e7-fe64830b613a">

Reply:
It's not turned on. I think it was one of my switches to VSCode. I added a `"files.trimTrailingWhitespace": true,` there